logging/logging.go

```diff
@@ -419,19 +420,34 @@ func (instance *Logging) sendErrorResponse(errorStr, logID string) {
 func (instance *Logging) addServiceIDFilter(journal *sdjournal.Journal,
```

Comment:
This is a strange function, isn't it? It accepts only two variants of fieldName. What if the field name is neither unitField nor serviceField? Also, the name doesn't perfectly match its functionality. Make two separate functions to add unit and service id filters.

Reply:
it was two separate function before and you asked merged them into one in previous set of comments. okay, i'll divide it into two